submissions/kyle-jeremy/scripts/article.js

```diff
@@ -0,0 +1,57 @@
+var articles = [];
+
+function Article (opts) {
+  // TODO: Use the js object passed in to complete this contructor function:
+  // Save ALL the properties of `opts` into `this`.
+  this.title = opts.title;
+  this.category = opts.category;
+  this.author = opts.author;
+  this.authorUrl = opts.authorUrl;
+  this.publishedOn = opts.publishedOn;
+  this.body = opts.body;
+}
+
+Article.prototype.toHtml = function() {
+  var $newArticle = $('article.template').clone();
+
+  $newArticle.attr('data-category', this.category);
+
+  // TODO: Use jQuery to fill in the template with properties
+  // from this particular Article instance. We need to fill in:
+  // the author name and url, the article title and body, and the
+  // publication date.
+  $newArticle.find('h1').text(this.title);
+  $newArticle.attr('data-time', this.publishedOn);
```

Comment:
Line 24 is close! You may have meant to chain the .attr method to a .find('time[pubdate]') - in this case, you may have also meant to declare attr('datetime') instead of 'data-time'